newlib/libm/ld/common/e_lgammal.c

```diff
@@ -11,14 +11,16 @@
  *
  */
 
+#include <sys/cdefs.h>
```

Comment:
This shouldn't be necessary -- this file gets included by ../e_lgammal.c which includes a bunch of headers.